### ISSUE TITLE
Overview of Aging pages is Missing in Documentation

### ISSUE THREAD

**Issue by nouman17610** [wagtail/guide#335]
The purpose of Aging pages is to report the pages on the basis of their age. You can see the list of pages from the latest to the latest pages.

![Aging-pages-Wagtail](https://github.com/wagtail/guide/assets/112282629/424a5b60-1904-4195-8092-8c1f90a134c3)

**Why it should be documented?**
The first-time user finds it difficult to understand the purpose of the Aging pages feature. The user needs a clear understanding of the purpose of the Aging Pages webpage. The overview of the page would help a user to avoid any confusion while managing the aging pages.
Clear guidelines to use filters increase efficiency and avoid confusion. Documentation should maintain consistency in providing clear guidelines about the pages and features.

**How it should be documented?**
There is a need for an overview of the purpose of the Aging pages feature. 
Furthermore, "How to use filters to see reports" guide is required.

**Comment by thibaudcolas** [comment]
@nouman17610 is this not already considered in https://guide.wagtail.org/en-latest/concepts/reports/#aging-pages-report ? Do you think there is more content needed?

**Comment by nouman17610** [comment]
@thibaudcolas 
You are right this is mentioned in the Reports Section. I believe that information is not sufficient to cover all elements on the page. This information is good for repeated users but first-time user needs clear and complete information regarding the purpose of the Aging Pages Report. 

It is important to add the following details to the Aging page guide.

- Overview of the Aging Pages
- The explanation about the download option.
- Use of filter. 
- Explanation of action to review pages.

Overall, existing content for Aging Pages doesn't provide a complete guide to do necessary actions for the Aging Pages. More details are required to equip the user with complete information.

**Comment by thibaudcolas** [comment]
I believe this is covered in the report page’s introduction:

> You can further filter what pages are displayed, and also export these reports in
